conans/test/functional/conan_api/two_conan_creates_test.py

```diff
@@ -37,11 +37,17 @@ def get_conaninfo(info):
                     return load(os.path.join(folder, "conaninfo.txt"))
 
                 settings = ["compiler=Visual Studio", "compiler.version=15", "build_type=Release"]
-                info = api.create(".", user="conan", channel="stable", settings=settings)
+                profile_host = ProfileInfo(None, settings, None, None)
+                profile_build = ProfileInfo(None, None, None, None)
```

Comment:
I think `profile_build` should be None

Reply:
I think it is better to keep it non defaulted to `None`. Here in the test we need this kind of initialization, but in the production code we always have a value and there is no need to introduce and additional if-clause to check if it is `None`.